src/main/resources/templates/search.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org/">
+<head th:replace="fragments :: head">
+</head>
+<body>
+
+<div class="container body-content">
+
+    <h1 th:text="${#strings.capitalizeWords(title)}"></h1>
+
+    <form th:action="@{/search/results}" method = "post">
+        <h2>Search by:</h2>
+
+
+        <p>
+            <span th:each="column : ${columns}">
+                <input
+                        type="radio"
+                        name="searchType"
+                        th:id="${column.key}"
+                        th:value="${column.key}"
+                        th:checked="${column.key == 'all'}"/>
+                <label th:for="${column.key}" th:text="${column.value}"></label>
+            </span>
+
+        </p>
+
+        <p>
+            <label for="searchTerm">KeyWord:</label>
+            <input type="text" name="searchTerm" id="searchTerm" />
+
+        </p>
+
+        <input type="submit" value="Search" />
+        <a href=" /adv-search">Advanced Search</a>
+
+    </form>
+
+    <hr />
+
+
+    <table class="recipe-listing">
+        <tr th:each="recipe : ${recipes}">
+            <td>
+                <p>
+                    Recipe Id:
+                    <a th:href="@{'/view/' + ${recipe.id}}" th:text="${recipe.id}"></a>
+                </p>
+                <p th:text="${'Name: ' + recipe.name}"></p>
+                <p>
+                    Name:
+                    <a th:href="@{'/names/view/' + ${recipe.name.id}}" th:text="${recipe.name}"></a>
+                </p>
+                <p>
+                    IngredientList:
```

Comment:
You can probably just display this by calling them "Ingredients" I only called the field "ingredientList" because I thought calling it just "ingredients" could get confusing with all the times I used the singular "ingredient" in the code.